samples/csharp/web/avatar/wwwroot/js/basic.js

```diff
@@ -0,0 +1,312 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
```

Comment:
Can you follow this change to log the WebRTC event?
https://github.com/Azure-Samples/cognitive-services-speech-sdk/commit/18032c07aecbdecf7ab1a83a9d7caed6a2111d26

Reply:
added on basic.js and chat.js